src/Timestamp.php

```diff
@@ -85,10 +85,14 @@ public function serialize(): array
 
     public function toJsonLd(): array
     {
-        $jsonLd = $this->serialize();
-        $jsonLd['@type'] = 'Event';
-
-        return $jsonLd;
+        return array_merge(
```

Comment:
This is indeed a safer approach then reusing the `serialize` method.

Reply:
Yeah I don't mind reusing it if it's the same anyway, but when they start to diverge like now we should split up the serialization & json encoding. It's usually easy enough to start with the same and then split up later though so there was no harm in re-using `serialize()` at first.